CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [3.20.0-beta.0] - 2019-12-11
+### Added
+- Custom CSS styles for `product-identifier`.
+
+## [3.20.0-beta] - 2019-12-06
+### Changed
+- Use new flexible `minicart.v2` and `add-to-cart-button`.
+
```

Comment:
Do we need to document changes on beta? I think we can just keep them on the unreleased section, but that's up to debate

Reply:
I'm not sure about that too hahaha
I could move these changes to `Unreleased` and release them in the new version.